src/shared/modules/connections/connectionsDuck.js

```diff
@@ -59,12 +60,14 @@ export const SWITCH_CONNECTION_FAILED = NAME + '/SWITCH_CONNECTION_FAILED'
 export const DISCONNECTED_STATE = 0
 export const CONNECTED_STATE = 1
 export const PENDING_STATE = 2
+export const CONNECTING_STATE = 'CONNECTING_STATE'
```

Comment:
@oskarhane I'm not really sure about this one. I decided to use a string to denote that it is not a driver status, but maybe that is confusing?

Reply:
Yes, please update to follow the same pattern as the other. Thanks for pointing it out.